src/main/java/net/revelc/code/formatter/xml/lib/FormattingPreferences.java

```diff
@@ -81,4 +86,14 @@ public void setSplitMultiAttrs(Boolean setSplitMultiAttrs) {
         if (setSplitMultiAttrs != null)
             this.splitMultiAttrs = setSplitMultiAttrs;
     }
+
+    public String getSaxValidation() { return saxValidation; }
+
+    public void setSaxValidation(String saxValidation) {
+        if (saxValidation.equals(IGNORE) || saxValidation.equals(FAIL)) {
+            this.saxValidation = saxValidation;
+        } else {
+            this.saxValidation = WARN;
```

Comment:
Defaulting to WARN silently even when receiving an unrecognized argument is probably not a very friendly API behavior. It should throw `IllegalArgumentException` if it doesn't recognize the argument.

Reply:
Good point.  I will fix this.